### PR TITLE
3 new mentors added - Busra, Gabriel and Fernanda

### PR DESCRIPTION
**Pull request by mads6879**
## Description
3 new mentors added - Busra, Gabriel and Fernanda

## Change Type
- [ ] Bug Fix
- [ ] New Feature
- [ ] Code Refactor
- [x] Mentor Update
- [ ] Documentation
- [ ] Other

## Screenshots
image

image

image

Total mentors = 40
image

## Pull request checklist

Please check if your PR fulfills the following requirements:
- [ ] I checked and followed the [contributor guide](CONTRIBUTING.md) 
- [ ] I have tested my changes locally.
- [ ] I have added a screenshot from the website after I tested it locally

### CODE REVIEW

**Comment by mads6879** [comment]
I have kept the sort order as 100 for all 3 of them for now. But when we open ad-hoc registrations for July, we can look at increasing their sort order to 200 so we can promote them as they are new and have 5 hours available (TBD).

File: _data/mentors.yml [hunk 1]
```diff
@@ -1237,3 +1237,119 @@
   network:
     - linkedin: https://www.linkedin.com/in/jyoti-y-b8232969/
     - medium: https://medium.com/@jyotiyadav99111
+
+- name: Busra Ecem Sakar
+  disabled: false
+  matched: false
+  sort: 100
```
Comment (dricazenck) [inline _data/mentors.yml]:
```suggestion
  sort: 200
```

File: _data/mentors.yml [hunk 2]
```diff
@@ -1237,3 +1237,119 @@
...
+      - Programming and Tools: SQL
+      - Data-Driven Decision Making
+      - Career Path Guidance
+  network:
+  - linkedin: https://www.linkedin.com/in/busraecemsakar/
+
+- name: Fernanda Lancao Morais Martins
+  disabled: false
+  matched: false
+  sort: 100
```
Comment (dricazenck) [inline _data/mentors.yml]:
```suggestion
  sort: 200
```

File: _data/mentors.yml [hunk 3]
```diff
@@ -1237,3 +1237,119 @@
...
+      DevOps practices, how to start in infrastructure, resume review, learn cloud, infrastructure as code, incident management, improve your pipeline, manage platform priorities, how to better support developers
+  network:
+  - linkedin: https://www.linkedin.com/in/flmmartins/
+  - twitter: https://x.com/flmmartins/
+  - github: https://github.com/flmmartins/
+
+- name: Gabriel Oliveira
+  disabled: false
+  matched: false
+  sort: 100
```
Comment (dricazenck) [inline _data/mentors.yml]:
```suggestion
  sort: 200
```

File: _data/mentors.yml [hunk 4]
```diff
@@ -1237,3 +1237,119 @@
...
+  position: QA Engineer, Picnic
+  bio: |
+    I am a pragmatic Agile tester dedicated to enhancing team deliveries through the development and maintenance of automated tests, fostering cross-disciplinary collaboration to establish a unified understanding of our products, and providing timely, actionable feedback to stakeholders.
+
+    🤔 You may be thinking "So what, how does that help me"? Or you may be in the category of "I don't know where to grow/learn next"? Book a session and tell me a bit about yourself so we can together find a way on how my experience can help you 😉
+
+    🥶 Also, if you get frozen solid when doing test automation, I also offer an guidance on automated testing where we can deep-dive into the code of your choice together.
+  image: 'assets/images/mentors/gabriel_oliveira.jpg'
+  languages: English, Portuguese
+  availability: []
```
Comment (dricazenck) [inline _data/mentors.yml]:
```suggestion
  availability: [ 7, 8 ]
```

File: _data/mentors.yml [hunk 5]
```diff
@@ -1237,3 +1237,119 @@
...
+  index: 40
+  location: Amsterdam/Netherlands
+  position: DevOps Engineer, Clay Solutions
+  bio: |
+    I work with infrastructure engineering for 13 years. I love all things infrastructure: designing platforms in the cloud with Azure/AWS, coding in terraform/ansible/python/shell.., building awesome delivery pipelines and helping developers.
+    I am from Brazil, currently living in Amsterdam and worked in several companies and different industries.
+    I am community-driven: I have presented at conferences, contributed to open source and like sharing knowledge.
+  image: 'assets/images/mentors/fernanda_martins.jpg'
+  languages: English, Portuguese
+  availability: []
```
Comment (dricazenck) [inline _data/mentors.yml]:
```suggestion
  availability: [ 7, 8 ]
```

File: _data/mentors.yml [hunk 6]
```diff
@@ -1237,3 +1237,119 @@
...
+  type: both
+  index: 39
+  location: London
+  position: Senior Data Analyst, Mall IQ
+  bio: |
+    I have 10 years of experience as a Data Analyst and dual master's degrees in
+    Biostatistics and Big Data & Data Science Technology. I am an expert in A/B testing, data analysis, statistical modelling, and machine learning, as well as SQL, Python, R, and other tools. My passion is using data to make important business decisions.
+  image: 'assets/images/mentors/busra_sakar.jpg'
+  languages: English, Turkish
+  availability: []
```
Comment (dricazenck) [inline _data/mentors.yml]:
```suggestion
  availability: [ 7, 8 ]
```

**Review by dricazenck — APPROVED** [review]

**Pull request merged**